src/typescript/azure-openapi-validator/rule.ts

```diff
@@ -21,6 +21,7 @@ interface ValidationMessage {
 }
 
 export interface Rule {
+  
```

Comment:
Why the empty line here?

Reply:
to test pipeline

Reply:
For testing pipeline. You can comments /azp run.  So you can revert the changes, it's not necessary.

Reply:
changed